Tevling/Components/ChallengeForm.razor

```diff
@@ -73,6 +73,26 @@
                         }
                     </div>
                 </div>
+                <h5 class="mb-3">Templates</h5>
+                <div class="mb-5">
+                    <select class="form-select mb-3" @onchange="e => LoadSelectedTemplate(e.Value)">
+                        <option value="">Select a template to load</option>
+                        @for (int i = 0; i < _templates.Count; i++)
+                        {
+                            <option value="@i">@_templates[i].Title</option>
+                        }
+                    </select>
+                    <div class="form-row">
+                        <label>
+                            <a class="btn btn-primary selected-activity-type py-1" type="button"
```

Comment:
The user should perhaps get some sort of feedback that the template was added. What do you think?